README.md

```diff
@@ -2,23 +2,64 @@
-message in the UI
+message in the UI.
+
+Additionally, this plugin provides warnings in the Airflow UI for the following scenarios:
+- When the current version will reach its end of life (EOL) in 30 days (default, configurable).
+- When the current version has already reached its end of life (EOL).
+
+This plugin also shows options to dismiss the EOL warnings for a configurable number of days (default is 7 days). There is also an option to completely disable the EOL warning.
 
 ## Settings
 
 This plugin looks at the following settings under the `astronomer` section of
 the Airflow config. The easiest way of setting this is via environment
-variables prefixed with `AIRFLOW__ASTRONOMER__`
+variables prefixed with `AIRFLOW__ASTRONOMER__`.
 
 - `update_check_interval`
 
   Number of seconds between each update check. Default 86400 (one day). Set to
-  0 to disable update checks
+  0 to disable update checks.
 
 - `update_check_timeout`
 
-  HTTP timeout for requesting update document. Default 60
+  HTTP timeout for requesting update document. Default is 60.
 
 - `update_url`
 
-  URL to request to find out about more udpates. Default to `updates.astronomer.io`
+  URL to request to find out about more udpates. Default to `updates.astronomer.io`.
+
+- `eol_warning_opt_out`
+
+  Sets whether to opt out of EOL warnings. The default is `"False"`.
+
+- `eol_dismissal_period_days`
+
+  Sets the period for dismissing EOL warnings. The default is 7 days.
+
+- `eol_warning_threshold_days`
+
+  Sets the threshold for showing EOL warnings. The default is 30 days.
+
+### Configuration
+
+The following environment variables can be set to configure the behavior of the Astronomer update checks and EOL warnings:
+
+- `AIRFLOW__ASTRONOMER__UPDATE_CHECK_INTERVAL`: Sets the interval for checking updates. Default is `86400` seconds (24 hours).
+- `AIRFLOW__ASTRONOMER__UPDATE_CHECK_TIMEOUT`: Sets the HTTP timeout for requesting the update document. Default is `60` seconds.
+- `AIRFLOW__ASTRONOMER__UPDATE_URL`: Sets the URL to request to find out about more updates. Default is `https://updates.astronomer.io`.
+- `AIRFLOW__ASTRONOMER__EOL_WARNING_OPT_OUT`: Sets whether to opt out of EOL warnings. The default is `"False"`.
+- `AIRFLOW__ASTRONOMER__EOL_DISMISSAL_PERIOD_DAYS`: Sets the period for dismissing EOL warnings. The default is `7` days.
+- `AIRFLOW__ASTRONOMER__EOL_WARNING_THRESHOLD_DAYS`: Sets the threshold for showing EOL warnings. The default is `30` days.
+
+### Example Usage
+
+To set these environment variables, you can add the following lines to your environment configuration:
+
+```sh
+export AIRFLOW__ASTRONOMER__UPDATE_CHECK_INTERVAL=86400
+export AIRFLOW__ASTRONOMER__UPDATE_CHECK_TIMEOUT=60
+export AIRFLOW__ASTRONOMER__UPDATE_URL="https://updates.astronomer.io"
+export AIRFLOW__ASTRONOMER__EOL_WARNING_OPT_OUT="False"
+export AIRFLOW__ASTRONOMER__EOL_DISMISSAL_PERIOD_DAYS=7
+export AIRFLOW__ASTRONOMER__EOL_WARNING_THRESHOLD_DAYS=30
```

Comment:
This is also duplicative imo.

Reply:
I was trying to give an example of how they can set it. But can remove it.